openid-connect-server-spring-boot/src/main/java/org/mitre/CryptoConfig.java

```diff
@@ -0,0 +1,51 @@
+package org.mitre;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+
+import org.mitre.jose.keystore.JWKSetKeyStore;
+import org.mitre.jwt.encryption.service.impl.DefaultJwtEncryptionAndDecryptionService;
+import org.mitre.jwt.signer.service.impl.DefaultJwtSigningAndValidationService;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.Resource;
+
+import com.nimbusds.jose.JOSEException;
+import com.nimbusds.jose.JWEAlgorithm;
+
+@Configuration
+public class CryptoConfig {
+	@Bean(name = "defaultKeyStore")
```

Comment:
Maybe use `@ConditionalOnMissingBean(JWKSetKeyStore.class)` same for all default services, so users can override them if they want to